guidelines/round-table.md

```diff
@@ -0,0 +1,49 @@
+## Data Together Weekly Roundtable Call
+
+Once a week on **Mondays 5-5:30pm**, a rotating facilitator will lead a round table discussion of a Data Together-related topic, challenge, or question. We use a standing [Zoom call link](https://edgi-video-call-landing-page.herokuapp.com/https://zoom.us/j/455833201): [zoom.us/j/455833201](https://edgi-video-call-landing-page.herokuapp.com/https://zoom.us/j/455833201) and [GitHub `roundtable` issues](https://github.com/datatogether/datatogether/labels/roundtable) to run each call. **Check out [how to host](#how-to-host)** before running your first call.
+
+### Objectives
+
+- Check-in on ongoing work and potentially surface larger issues that need to be discussed in-depth
+- Allow initial feedback on developing ideas prior to opening a decision
+- Maintain momentum going between larger coordinating calls
+
+### Format
+
+Meetings will rotate individuals leading a round table around a specific topic, problem, or task. Scheduling topics will interleave implementation, governance, and topic area questions. Round table topics will be picked at least a week in advance.
```

Comment:
not totally clear whether rotation is in topics or individuals or both

Reply:
> Scheduling topics will interleave implementation, governance, and topic area questions.

I'm sorry if I'm being dense, but I don't know what this sentence means!

Reply:
Might this work? Sorry, hard to know if it's any better, but my mind stumbled on "rotate individuals leading a round table around" :)

> Meetings will rotate leaders for each round table on a specific topic, problem, or task.

Reply:
thanks, yeah this was messy. Hopefully better!